app/views/income_employment_status/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :header_title, 'Money & Income' %>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="step-section-header">
+      <div class="step-section-header__subhead">
+        Who in your household is currently employed, or has been in the past
+        30 days?
+      </div>
+      <p class="text--help step-section-header__subhead-help">
+        Employment includes temporary or contract jobs. Self employment
+        includes odd jobs, home businesses, online businesses, etc.
+      </p>
+    </div>
+  </header>
+
+  <div class="form-card__content">
+    <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
+      <% @step.members.each do |member| %>
+        <%= f.fields_for('members[]', member, hidden_field_id: true) do |ff| %>
+          <div class="household-member-group" data-md5="<%= data_md5(header_name(member))%>">
```

Comment:
it's basically just for the tests :) I was trying for about 30 min to target these elements based on the header name rather than some random attr, but it was not working. This is how the prototype had this implemented, so I just went with the "copy what they did" approach.

That being said, I can try to get it to work by selecting a section based on the h4 element again if you think this is too janky